New Sheet Submission

dedurrett opened the pull request:
Ironsworn RPG Character Sheet

Solarswordsman commented:
Hi @dedurrett, I'm personally super hyped to get an Ironsworn character sheet on here, I've been playing it myself and was definitely _this close_ to making one over the weekend.

That said, I can't accept this PR as-is; please remove the ACKS sheet from this pull request and put it in a separate PR to be reviewed.  Normally I would make an exception but this is a complete overhaul of a sheet someone else very recently authored and indicated that they wanted to improve upon, so it needs to undergo some further, detailed review.

Thanks,
Jeff

dedurrett commented:
Thanks Jeff!
I'm still new to this github stuff, so it was a mistake that these both ended up in the same request in the first place. Hopefully I corrected it? I just reverted the last commit which was the ACKS related stuff. Let me know if I missed any steps.

Solarswordsman commented:
Looks good!  Also, whoa, you must be quite an early user to have such a low user id.  Glad you're still with us! :D  Can't wait to play some Ironsworn on Roll20.  I'll merge this in and hopefully get it live soon.

Jeff
